tcms/testplans/static/testplans/js/get.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function() {
+    const testPlanId = $('#test_plan_pk').data('testplanPk');
+    const testCases = {
+        'confirmed': [],
+        'not_confirmed': []
+    };
+
+    jsonRPC('TestCase.filter', [{'plan': testPlanId}], function(data) {
+        for (var i = 0; i < data.length; i++) {
+            var testCase = data[i];
+            if (testCase.case_status_id === 2) {
+                testCases.confirmed.push(data[i]);
+            } else {
+                testCases.not_confirmed.push(data[i]);
+            }
+        }
+        drawConfirmedTestCases();
+
+    });
+
+    function drawConfirmedTestCases() {
+        var container = $('#confirmed-testcases'),
+            noCasesTemplate = $('#no_test_cases'),
+            testCaseRow = $('#test_case_row');
+
+        if (testCases.confirmed.length > 0) {
+            for (var i = 0; i < testCases.confirmed.length; i++) {
+                var testCase = testCases.confirmed[i];
+                testCaseRow[0].content.querySelector('.list-view-pf-main-info').innerText = `TC-${testCase.id}: ${testCase.summary}`;
+                container.append($(testCaseRow[0]).html());
+            }
+        } else {
+            container.append(noCasesTemplate[0].innerHTML);
+        }
+
+        treeViewBind();
+
+        $("#confirmed-testcases .list-group-item-container .close").on("click", function() {
+            $(this).parent().addClass("hidden")
+                .parent().removeClass("list-view-pf-expand-active")
+                .find(".fa-angle-right").removeClass("fa-angle-down");
+        });
```

Comment:
You don't need a close button. `treeViewBind()` takes care of opening & closing the expanded element once you click on the row representing it. So the X button is duplicating functionality and is not needed at all.